### DISCUSSION TITLE
iPhone 14 Pro系列規格全面進化 「動態島」功能超實用

### DISCUSSION THREAD

**Discussion by Valentinaa0214** [1101-GS4719-C/Python-Community-GS4719-C#686] [科技新知]
在發表會中壓軸亮相的iPhone 14 Pro和iPhone 14 Pro Max，差別僅在螢幕尺寸與電池續航力，最大亮點就是全新設計的「動態島」互動功能，並未如外媒預測出現驚嘆號設計，而是透過「動態島」模糊了軟硬體界線，能即時調整變化以顯示重要的提示、通知和活動，而不需要切換原本螢幕上正在操作的內容。而這次也首次於iPhone顯示器上帶來「永遠顯示」功能，讓新的「鎖定畫面」變得更實用，一眼便能掌握時間、小工具和「即時動態」。
1TB的要五萬二...

聯合新聞網:https://udn.com/news/story/7098/6597091?utm_source=linemobile&utm_medium=share

**Comment by Yunhans** [comment]
比瀏海好看耶

**Comment by bonny123865** [comment]
可惜我還是繼續用iPhone11，等到i17的時候再換手機!
sorry在再不分

**Reply by Yunhans** [reply]
再啦

**Comment by danwan1027** [comment]
對於好幾年沒換換手機的人，iPhone14 是個很好的選擇！

**Reply by Yunhans** [reply]
買給我買給我

**Reply by Chiway1229** [reply]
如果爸爸能借錢給我 我就能買了

**Comment by Chiway1229** [comment]
我記得在兩天前的新聞，小米已經仿造出動態島的盜版-動態鳥了
現代手機市場競爭真是激烈

**Comment by a45320614** [comment]
隨著蘋果推出「動態島」的功能，是否又會帶起一波其他手機廠模仿的風暴呢?
不過個人倒是覺得這個功能有點雞肋，跟瀏海相比也只是半斤八兩

**Comment by williamyen0623** [comment]
i14就像i12一樣, 是前一代擠牙膏出來的產品, 更新的東西不多, lighting沒換成type-c, 還有醜到爆的糯米腸, 還是瀏海好看～

**Comment by manipulateed** [comment]
i14軟硬體的深度整合，也算是對於技術的一次更新，也在多功能上更加貼近使用著的需求，他在創造需求讓我們看見。

**Comment by AnAnistired** [comment]
小知識:仔細看，iPhone14pro前鏡頭部分其實是驚嘆號，點和線之間中間有螢幕，但不知道為什麼蘋果軟體部分要把中間連在一起

**Comment by dinosaur1009** [comment]
一般版iPhone 14仍沿用舊款晶片....但pro的話價格差異好大.....

**Comment by harrylee1971** [comment]
PRO的話感覺真的是很破，強迫人去買PRO擁有新的性能

**Comment by zhiron544** [comment]
我覺得iPhone 14很吸引人，有創新和實用的功能，對「動態島」功能感興趣。

**Comment by Alex11392** [comment]
我真的好喜歡動態島，加入這個功能後更喜歡了
